### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of null

### ISSUE THREAD

**Issue by johngeorgewright** [atom/release-notes#25]
Uncaught TypeError: Cannot read property '0' of null

**Atom Version**: 0.161.0
**System**: linux 3.13.0-44-generic
**Thrown From**: [release-notes](https://github.com/atom/release-notes) package, v0.39.0

### Steps To Reproduce

I was trying to find a way of updating atom.

1. ctrl+shift+p
2. "Window: Update Available"

### Stack Trace

```
At /usr/share/atom/resources/app/node_modules/release-notes/lib/main.js:34

TypeError: Cannot read property '0' of null
  at atom-workspace.<anonymous> (/usr/share/atom/resources/app/node_modules/release-notes/lib/main.js:34:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/usr/share/atom/resources/app/node_modules/command-palette/lib/command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/usr/share/atom/resources/app/node_modules/autocomplete/node_modules/atom-space-pen-views/lib/select-list-view.js:335:21)
  at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom/resources/app/node_modules/autocomplete/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:167:20)

```

/cc @atom/core

**Comment by kevinsawicki** [comment]
This looks like a duplicate of https://github.com/atom/release-notes/issues/21 which should be fixed in the latest release, 0.174

**Comment by 50Wliu** [comment]
Linux has no way of auto-updating currently.  You'll need to re-download the debian package each time an update comes out.
